unit/solvers/lowering/byte_operators.cpp

```diff
@@ -215,7 +215,7 @@ SCENARIO("byte_extract_lowering", "[core][solvers][lowering][byte_extract]")
 
     THEN("byte_extract lowering yields the expected value")
     {
-      for(const auto endianness :
+      for(const auto &endianness :
           {ID_byte_extract_little_endian, ID_byte_extract_big_endian})
```

Comment:
FWIW this case is marginal because copying IDs is of course very cheap by design, but clang still warns about it regardless...